pynetbox/api.py

```diff
@@ -159,3 +160,30 @@ def __init__(
         self.tenancy = App(self, "tenancy")
         self.extras = App(self, "extras")
         self.virtualization = App(self, "virtualization")
+
+    def api_version(self):
```

Comment:
Ultra minor nit, but I think just `version()` is a good name since we're already in the `api` object.

Reply:
Renamed that. Btw, should we also make `version` a property (with the decorator)?

Reply:
:+1: Yeah, that's a good idea.